index.js

```diff
@@ -0,0 +1,21 @@
+fetch('https://api.github.com/users/TetianaKlitna/repos')
```

Comment:
For the open api project assignment, you should be querying one of the open source api options listed.  Eg. [Open-Meteo](https://open-meteo.com/) - a weather API
[Swapi.Tech](https://www.swapi.tech/) - an API about Star Wars films
[Marvel](https://developer.marvel.com/) - an API about the Marvel fandom
[ARTIC](https://api.artic.edu/docs/#introduction) - an art API from the Art Institute of Chicago
[TheDogAPI](https://thedogapi.com/) or the [TheCatAPI](https://thecatapi.com/) - APIs about (you guessed it!) Dogs or Cats
[SampleAPIs](https://sampleapis.com/api-list/coffee) - an API for coffee lovers

See the lesson assignment for open api project.

Reply:
I've added a weather api